collections/the university of manchester/Persian_MS_300.xml

```diff
@@ -164,13 +167,16 @@
                                             <persName type="standard" ref="http://viaf.org/viaf/56454583/#Navabpour,_Reza" key="person_56454583" role="cataloguer" change="#role-assignment">Reza Navabpour</persName> circa
                                             <date calendar="#Gregorian" notBefore="1992" notAfter="1994">1993</date>. Identification of
                                         provenance based on manuscript catalogue by Michael Kerney,
-                                            <date calendar="#Gregorian" notAfter="1901" notBefore="1868">circa 1890s.</date>
+                                            <date calendar="#Gregorian" notAfter="1901"
```

Comment:
This seems strange as well, to break the line on the `notBefore`. If it were a line-length issue then I could see it, but L167 above is still much longer. Is there a reason why the linebreak was introduced here, or is this a problem with an automated formatting script being too vigorous?